### DISCUSSION TITLE
Exporter filtering question on matching metrics with chart_context

### DISCUSSION THREAD

**Discussion by BolverBlitz** [netdata/netdata#18817] [Q&A]
Problem:
So i only want some charts, i´ve checked netdata frontend (The thing running port 19999) and wrote down all data i need, however the " send charts matching" seems to match the "chart_id" field and this should be unique. 

So so for disks this is fine, disk.sd* would match all my sda - sdag drives. But for other like S.M.A.R.T values this gets complicated the ID is like "smartctl.device_sdx_type_sat_smart_attr_command_timeout_normalized" so when i do "smartctl.device_sd*" i would get a lot of metrics i dont want. Kinda the same issue comes with "net.packets" there is also the interface name + port in the name.

Now i read that this may support regex, while this would work i would still have to extra check all of my diffrent host systems if i didn´t forget some exotic network card.

Question: Can i filter on the "chart_context" value, or is there another solution to my problem i´ve not seen yet?
Idearly i would like to filter like the web version on this value:
![grafik](https://github.com/user-attachments/assets/00307cf3-134c-4074-b838-2685d3dd4390)

For now i´ll filter this on my server side, but i would like to not send data over the network when i throw out 90%

```conf
[exporting:global]
    enabled = yes
    # send configured labels = yes
    # send automatic labels = no
    # update every = 10

[json:http:ebg_json_exporter]
    enabled = yes
    destination = 192.168.0.80:3001
    data source = average
    send obsolete charts = yes # Just for debugging
    send charts matching = system.cpu system.load system.ram mem.swap system.net net.packets disk.io disk.ops disk.await disk.space cpufreq.cpufreq smartctl.device_smart_status smartctl.device_ata_smart_error_log_count smartctl.device_power_on_time smartctl.device_temperature sensors.chip_sensor_power_average sensors.chip_sensor_temperature systemd.service
    hostname = some-server-name
    update every = 10
```

**Comment by m0hammadjaan** [comment]
@BolverBlitz I am facing the same issue. In my case, I have setup a central netdata server and other child hosts are sending data to it. The problem is that, on the dashboard I am not able to get the latest data and metric values for most of the hosts. I guess it may be because the central server is getting the data at high rate from all the server, I thought restrict the streaming of metrics may help me in this case. Let me know if you find the solution to your problem.
